validphys2/src/validphys/theorycovariance/construction.py

```diff
@@ -363,7 +370,9 @@ def covs_pt_prescrip(
             deltas1 = list((other - central1 for other in others1))
             central2, *others2 = process_info.theory[name2]
             deltas2 = list((other - central2 for other in others2))
-            if l == 3:
+            if point_prescription in ["alpha_s", "alpha_s_reduced", "alpha_s_extended"]:
+                s = covmat_alphas(name1, name2, deltas1, deltas2)     
```

Comment:
I guess this should go inside l==3 as is aldo done in theorycovarianceutils.py. 

I understand you probably quickly coded this up and we may change this PR anyway, but it's just something I noticed and wanted to make a note of